package.json

```diff
@@ -3,14 +3,18 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@afc-org/react-tailwind": "^0.1.1",
```

Comment:
what is this library exactly do for you here? as I see you are using the modal from `headlessui` library